package.json

```diff
@@ -35,15 +35,15 @@
     "NOTICE"
   ],
   "scripts": {
-    "lint": "eslint index.js lib/**/*.js test/*.js test/**/*.js",
+    "lint": "eslint --fix index.js lib/**/*.js test/*.js test/**/*.js",
     "test": "npm run lint && jest --runInBand",
     "test-watch": "npm run lint && jest --runInBand --watch",
     "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha -- test/*.js test/**/*.js",
-    "preversion": "npm test",
-    "version": "node ./scripts/version.js && npm run generate-docs",
```

Comment:
Could we still call the `./scripts/version.js` file somehow in the release? All that file is doing is updating the version number of style dictionary in the examples to be the new version number. Or if there is a better way to do this lets do that.

Reply:
Calling it and doc generation after version bump.
A suggestion would be to create a monorepo with lernajs and make examples a package that depends on style-dictionary. When bumping SD, all others would be updated. But this is another task

Reply:
you can call `npm run release -- --dry-run` to test things out. It will run the postbump though.

Reply:
Ohhh using lerna is a good idea!